cfgov/unprocessed/apps/analytics-gtm/browserslist-stats.json

```diff
@@ -0,0 +1,288 @@
+{
```

Comment:
This is an auto-generated file from browserlist-ga

Reply:
Be sure to ignore it in the linter

Reply:
Should be okay since it's json?

Reply:
Oh, probably yeah.